src/lib/sys.c

```diff
@@ -68,6 +57,14 @@ int run_command_full_params(const char *stdout_file, const char *stderr_file, ch
 
 	// Main
 	if (pid > 0) {
+		close(pipe_fds[1]);
+		char *line = NULL;
+		size_t len = 0;
+
+		while ((getline(&line, &len, pipe_fds[0])) != -1) {
```

Comment:
getline can't be used with a file descriptor. You need to use read() here.

Reply:
Ya i realized that..got a mighty seg fault 🗡

Reply:
done